app/views/member_mailer/welcome_coach.html.haml

```diff
@@ -6,7 +6,11 @@
 
 %p
   - if @member.subscriptions.any?
-    You'll automatically receive invitations to our workshops via email.
+    For our London chapter, you will be able to RSVP to workshops via our website. We open RSVPs at 1PM on the Friday before the workshop. You will be able to register through the workshop page. Join the #london-chapters channel
+    = link_to 'on our Slack', 'http://codebar-slack.herokuapp.com/'
+    to receive a notification when we open RSVPs, or follow us on Twitter
+    - link_to '@codebar', 'https://twitter.com/codebar'
+    . For our other chapters, you'll automatically receive invitations to our workshops to your inbox.
   - else
```

Comment:
`to your inbox` seems a little weird, could we just say `via email`?

Reply:
sure, can update